jetson/percep/pcl.cpp

```diff
@@ -280,26 +280,41 @@ double PCL::FindClearPath(const std::vector<std::vector<int>> &interest_points,
     
     //Check Center Path
     if(CheckPath(interest_points, viewer, obstacles, compareLine(0,-HALF_ROVER), compareLine(0,HALF_ROVER))) {
-        std::cout << "CENTER PATH IS CLEAR!!!" << std::endl;
+            std::cout << "CENTER PATH IS CLEAR!!!" << std::endl;
         return 0;
     }
+    else { //otherwise, check to find left and right paths
+
+        //Values that store the distances of the last obstacle from a given CheckPath
+        double centerDistance, leftDistance, rightDistance;
+        centerDistance = distance; //from previous loop of CheckPath, where it was only looking at the center path for a clear path
```

Comment:
May want to consider initializing centerDistance in the same line as the others are declared (289) just for efficiency and clarity